database/frontend/icons.json

```
@@ -237,11 +237,16 @@
     "category": "frontend",
     "subcategory": "icons"
   },
+  {
+    "name": "Freepik Icons",
+    "description": "The biggest free icon library in the world. Find icons that go together Fast. It has 10M+ ready-to-use, free license icons.",
+    "url": "https://www.freepik.com/icons",
```

[proofreading] Don't forget to add the `category` and `subcategory` names. Refer to [this section of our Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md#appending-new-links-) for more context. 
```suggestion
    "name": "Freepik Icons",
    "description": "The biggest free icon library in the world. Find icons that go together Fast. It has 10M+ ready-to-use, free license icons.",
    "url": "https://www.freepik.com/icons",
    "category": "frontend",
    "subcategory: "icons"
```